sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -5607,18 +5612,14 @@ pi_result piEventRetain(pi_event Event) {
 }
 
 pi_result piEventRelease(pi_event Event) {
-  return EventRelease(Event, nullptr);
-}
-
-static pi_result EventRelease(pi_event Event, pi_queue LockedQueue) {
   PI_ASSERT(Event, PI_INVALID_EVENT);
   if (!Event->RefCount) {
     die("piEventRelease: called on a destroyed event");
   }
 
   if (--(Event->RefCount) == 0) {
     if (!Event->CleanedUp)
-      Event->cleanup(LockedQueue);
+      cleanup(Event);
```

Comment:
It looks like this call to "cleanup" here is wrong, or at lease redundant.
Since refcount came to 0 already, it means the "cleanup" was already done?
What do you think?